cmd/pipeline/controller.go

```diff
@@ -404,10 +409,20 @@ func getFilteredProwJobs(id string, pjsToFilter []*prowjobv1.ProwJob) []*prowjob
 // it can also abort the given if it is running and has the same identifier as the detected
 // newer prowjob. The function returns the updated prowjob if it was aborted, otherwise the
 // original prowjob is returned.
+//
+// When AllowConcurrentPostsubmitJobs feature flag is enabled, postsubmit jobs are allowed
+// to run concurrently and will not be aborted. Presubmit jobs will still be aborted.
```

Comment:
I'm not too keen on having a method named `abortDuplicatedProwjobs` that sometimes does not abort duplicated prowjobs. Could we do the check at the callsite and conditionally call or not call the `abortDuplicatedProwjobs` instead?